apps/site-nft/src/components/Footer/styles.module.less

```diff
@@ -0,0 +1,20 @@
+@import "~@thecointech/site-semantic-theme/variables.less";
+
+#footerContainer{
+  grid-area: footer;
```

Comment:
Look! The footer requests where it should be displayed, and the app pops it in there.

No need to worry about positioning, it's parents or anything like that!  It's really quite clean!